### ISSUE TITLE
Locate grain function is slow due to re-running grain finding

### ISSUE THREAD

**Issue by rhysgt** [MechMicroMan/DefDAP#129]
Even after the `find_grains` function has been executed succesfully, the function executes again when using the `locate_grain` function, wasting a lot of time.

**Comment by mikesmic** [comment]
Is this on the dev branch I assume? Running a function directly will not save to the datastore. You have to 'generate' the data  you want, for grains you would run:
```ebsd_map.data.generate('grains', min_grain_size=10)```

**Comment by mikesmic** [comment]
`locate_grain` should generate the grains automatically, so it should have only run at the first execution. I'll have a look at that

**Comment by rhysgt** [comment]
Yes on develop. Ah ok, maybe we should change the example notebook to use generate instead of find_grains?

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by mikesmic** [comment]
It is updated in the main example notebook, not sure about the others
